modules/bookmark.js

```diff
@@ -0,0 +1,31 @@
+const Augur = require("augurbot"),
+  u = require("../utils/utils");
+
+// Message context menu for bookmarking a message.
+
+const Module = new Augur.Module()
+.addInteractionCommand({name: "Bookmark",
+  commandId: undefined,
```

Comment:
Will this be a guild-level command? If so, it needs the `guildId` attribute. Disregard if global.

Reply:
Intended to be bot-wide.